exercises/allergies/HINTS.md

```diff
@@ -6,5 +6,6 @@ with `Eq` and `Show` instances, and implement the following functions:
 - `allergies`
 - `isAllergicTo`
 
-Your can use the provided signatures if you are unsure about the types, but
-don't let them restrict your creativity.
+You will find a dummy data declaration and type signatures already in place,
```

Comment:
If this is *too* different from the original ("Don't let them restrict your creativity"), please help me to combine the two in a sensible manner.